gym/spaces/box.py

```diff
@@ -232,7 +232,7 @@ def to_jsonable(self, sample_n):
         """Convert a batch of samples from this space to a JSONable data type."""
         return np.array(sample_n).tolist()
 
-    def from_jsonable(self, sample_n: Sequence[SupportsFloat]) -> List[np.ndarray]:
+    def from_jsonable(self, sample_n: Sequence[Union[float, int]]) -> List[np.ndarray]:
```

Comment:
Why this restriction? I think SupportsFloat is more general?

Reply:
Given that it should be from json then we know it must be a float or int but you are correct that supports float is a more general type